lib/tty/prompt/list.rb

```diff
@@ -177,7 +177,11 @@ def arrows_help
       #
       # @api private
       def key_help_label(key_name)
-        %i[return enter].include?(key_name) ? "Enter" : key_name.to_s.capitalize
+        if %i[return enter].include?(key_name)
+          "Enter"
+        else
+          key_name.to_s.split("_").map(&:capitalize).join("+")
```

Comment:
I thought this was a sensible default, as it covers nicely printing most ctrl/shift+ combination keys. The custom labels should allow the user to deal with more thorny cases.